config/templates/cloud-config-controller

```diff
@@ -693,6 +705,66 @@ write_files:
            $kubectl uncordon "$hostname"
            echo "done."'
 
+  {{if .Experimental.Plugins.Rbac.Enabled }}
+
+  #https://github.com/coreos/kube-aws/issues/230#issuecomment-274946933
+  - path: /srv/kubernetes/rbac/clusterrole.yaml
+    content: |
+        kind: ClusterRole
+        apiVersion: rbac.authorization.k8s.io/v1alpha1
+        metadata:
+            name: full-access
+        rules:
+          - apiGroups: ["*"]
+            resources: ["*"]
+            verbs: ["*"]
+          - nonResourceURLs: ["*"]
+            verbs: ["*"]
+
```

Comment:
@gianrubio won't that clash with the same-named cluster-admin role created the the RBAC bootstrap? Or will one just replace the other?

Do people use new versions of kube-aws to create old k8s version clusters? Or can we just support from the version that is associated with each kube-aws release? @mumoshu ?